lib/services/users.pm

```diff
@@ -43,13 +43,16 @@ sub lock_screen {
 }
 
 sub logout_and_login {
+    my ($login_user, $login_pw) = @_;
+    my $test_user = $login_user // $username;
+    my $test_pw   = $login_pw   // $newpwd;
     handle_logout;
     send_key_until_needlematch 'displaymanager', 'esc', 9, 10;
     mouse_hide();
     wait_still_screen;
-    assert_and_click "displaymanager-$username";
+    assert_and_click "displaymanager-$test_user";
```

Comment:
Do you need add new needle at here?

Reply:
No, the needle displaymanager-test already in openQA.